src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -171,5 +178,9 @@ public class LoginPasswordlessDetails
         public string DeviceType { get; set; }
 
         public string IpAddress { get; set; }
+
+        public bool? Approved { get; set; }
+
+        public DateTime? ResponseDate { get; set; }
```

Comment:
I don't see these properties being used anywhere; just being set. Can they be removed?

Reply:
Yes they can, initially i was passing them on constructor of the page, but them realised i had to call the method to get the login request on button click.